Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -616,12 +630,19 @@ void RomBrowserWidget::on_RomBrowserThread_Finished(bool canceled)
     else
     {
         // use settings' width
-        for (int i = 0; i < (columnSizes.size() - 1); i++)
+        for (int i = 0; i < columnSizes.size(); i++)
```

Comment:
Why did you change this if I may ask? the last column should always be stretched, otherwise the width of the last column looks weird, no?

Reply:
Keeping the last column stretched causes weird issues when moving the columns around. A lot of the time, one of the first two columns stays stretched instead of the last one depending on how they're arranged when the layout is saved. It's pretty late in my time zone right now and I'm about to go to bed, but I'll try and see if I can figure out a better solution for that once I wake up.